src/lib/tabs/tab-group.spec.ts

```diff
@@ -169,6 +176,22 @@ describe('MdTabGroup', () => {
       expect(testElement.querySelectorAll('.mat-ripple-element').length)
         .toBe(0, 'Expected no ripple to show up on label mousedown.');
     });
+
+    it('should switch tabs on body swipe', () => {
```

Comment:
Also need tests for skipping disabled tabs and for swipes at the first/last tab.